exercise_0/fileMode/FileMode.h

```diff
@@ -0,0 +1,14 @@
+//
+// Created by emyrmak on 7/3/2019.
+//
+#pragma once
+#include <string>
+#include "../characterCounter/CharacterCounter.h"
```

Comment:
Like I said previously, please avoid relative includes. Check out this comment: https://github.com/Maksysko/C-training/pull/1#discussion_r300368297